charts/canvas-oda/README.md

```diff
@@ -1,6 +1,6 @@
 # canvas-oda
 
-![Version: 1.1.8-lt4](https://img.shields.io/badge/Version-1.1.8--lt4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1beta4](https://img.shields.io/badge/AppVersion-v1beta4-informational?style=flat-square)
+![Version: 1.1.8-lt4](https://img.shields.io/badge/Version-1.1.8--lt4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1](https://img.shields.io/badge/AppVersion-v1-informational?style=flat-square)
```

Comment:
Should be v1.2.0 here?

Reply:
These are auto-generated README files generated by `helm-docs` tool. I've regenerated all the README's